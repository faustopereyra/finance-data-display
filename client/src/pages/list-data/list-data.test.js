import { shallow } from "enzyme";
import React from "react";
import ListData from "./list-data.component";

it("expect to render ListData component", () => {
    expect(shallow(<ListData />)).toMatchSnapshot();
})