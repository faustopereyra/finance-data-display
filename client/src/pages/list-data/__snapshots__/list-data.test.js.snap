// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render ListData component 1`] = `ShallowWrapper {}`;
